src/utils.ts

```diff
@@ -647,6 +648,28 @@ function isValidSnapshot(snapshot: number | string, network: string) {
   );
 }
 
+function isStarknetAddress(address: string): boolean {
+  if (!address) return false;
+
+  try {
+    validateAndParseAddress(address);
+    return true;
+  } catch (e: any) {
+    return false;
+  }
+}
+
+function getFormattedAddress(
+  address: string,
+  format: string[] = ['evm', 'starknet']
+): string {
+  if (format.includes('evm') && isAddress(address)) return getAddress(address);
+  if (format.includes('starknet') && isStarknetAddress(address))
+    return validateAndParseAddress(address);
+
+  throw new Error(`Invalid address: ${address}`);
+}
```

Comment:
all other functions in this file are exported in `export default` and individually too, I mean for example 
```
export async function getVp(
export async function getJSON
etc..
```

I don't remember exactly where, but it will fail to import in some environment if we don't have that. maybe better if we add that like others?